### ISSUE TITLE
Vizy fields not searchable

### ISSUE THREAD

**Issue by Mosnar** [verbb/vizy#1]
**Description**

It doesn't look like any text content is being added to the search index, so the content is not discoverable by search.

**Steps to reproduce**
1. Create Vizy field with search enabled
2. Add content and see that the search index is not updated.

**Additional info**
- Plugin version: 1.0.0
- Craft version: 3.7-dev
- Multi-site: No

**Comment by engram-design** [comment]
I believe only the base-level content was added to the index, not the fields within Vizy blocks, or within nested Vizy fields.

Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.0",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.1](https://github.com/verbb/vizy/releases/tag/1.0.1)
